pkg/spec/config_linux.go

```diff
@@ -94,6 +94,22 @@ func addDevice(g *generate.Generator, device string) error {
 	if err != nil {
 		return err
 	}
+
+	if rootless.IsRootless() {
+		perm := "ro"
+		if strings.Contains(permissions, "w") {
+			perm = "rw"
+		}
+		devMnt := spec.Mount{
+			Destination: dst,
+			Type:        TypeBind,
+			Source:      src,
```

Comment:
Should probably check if src exists?  would get better error then if it happens in the OCI.

Reply:
fixed in the new version